app/common/controllers/intercom.js

```diff
@@ -0,0 +1,28 @@
+module.exports = [
+    '$scope',
+    '$rootScope',
+    'UserEndpoint',
+function (
+    $scope,
+    $rootScope,
+    UserEndpoint
+) {
+    $rootScope.$on('event:authentication:login:succeeded', function () {
+        UserEndpoint.getFresh({id: 'me'}).$promise.then(function (user) {
+            window.Intercom('boot', {
+                app_id: window.ushahidi.intercomAppId,
+                email: user.email,
+                created_at: user.created,
+                user_id: user.id,
+                'realname' : user.realname,
+                'last_login': user.last_login,
+                'role': user.role,
+                'logins': user.logins
```

Comment:
Most of the tracking can be a future issue. But ideally the user -> deployment link would go in now.. not sure how much work is involved though?